apps/core-app/src/components/ShamanInput.tsx

```diff
@@ -0,0 +1,10 @@
+import { Select } from '@daohaus/ui';
```

Comment:
Let's move this into the UI library as a moecule

Reply:
Will do